### PR TITLE
Fixed hook locations of twig files

### LINKED ISSUE #21475: Not functioning hook AdminProductsMainStepLeftColumnBottom
#### Describe the bug

The hook AdminProductsMainStepLeftColumnBottom is not working.
According to the docs it should be located in: /src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig
But it's located in: Located in: /src/PrestaShopBundle/Resources/views/Admin/Product/PorductPage/Panels/essentials.html.twig

#### Expected behavior

When registering the hook in module like so:
$this->registerHook('displayAdminProductsMainStepLeftColumnBottom')

It should display whatever is return like so:
public function hookAdminProductsMainStepLeftColumnBottom(){
}

#### Additional information

* PrestaShop version: 1.7
* PHP version: N/A

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Fixed location of twig files
| Fixed ticket? | Fixes PrestaShop/PrestaShop#21475

### CODE REVIEW

File: src/content/1.7/modules/concepts/hooks/list-of-hooks.md [hunk 1]
```diff
@@ -1333,7 +1333,7 @@ displayAdminGridTableBefore
 
     This hook adds new blocks before Grid component table.
 
-    Located in: src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Blocks/table.html.twig
+    Located in: templates/bundles/PrestaShopBundle/views/Admin/Common/Grid/Blocks/table.html.twig
```
Comment (PierreRambaud) [inline src/content/1.7/modules/concepts/hooks/list-of-hooks.md:1336]: Maybe add a note at the begin of this file to tell something like:
"Before the 1.7.9.0, files were inside the `src/PrestaShopBundle/Resources` directory"

**Review by eternoendless — COMMENTED** [review]
This shouldn't be done in 1.7 docs

**Comment by matks** [comment]
This PR needs to target the `8.x` branch that will soon be created

File: src/content/1.7/modules/concepts/hooks/list-of-hooks.md (hunk 1 above)
Comment (matks) [reply src/content/1.7/modules/concepts/hooks/list-of-hooks.md]: Now that we have a documentation section for 1.7 and a documentation section for 8, this is not needed anymore I believe 😉

**Review by matks — APPROVED** [review]

**Review by matks — COMMENTED** [review]
@Progi1984 I approve the PR :) the changes are valid IMO. However it needs a rebase 😞

**Comment by Progi1984** [comment]
@matks Thanks. Rebase is done.

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @Progi1984

**Pull request merged**